x-pack/plugins/security_solution/common/ecs/file/index.ts

```diff
@@ -14,9 +14,30 @@ export interface CodeSignature {
   subject_name: string[];
   trusted: string[];
 }
+
+export interface Token {
+  integrity_level_name: string;
+}
+
+export interface MemoryPe {
```

Comment:
what does `Pe` suffix mean?  Can we spell it out instead of shorten it up (if possible)? (same for other areas below where this suffix is used)

Reply:
I am not sure what it means, maybe "portable executable". Everywhere inside the indexes is mentioned as PE. @kevinlog do you know?

Reply:
Yes it's a [Portable Executable](https://github.com/elastic/endpoint-package/blob/1c983ab3fe1ebaa62171b38cb37c3dc420d9b16b/custom_schemas/custom_pe.yml#L5-L13) file that's [found in memory](https://github.com/elastic/endpoint-package/blob/1c983ab3fe1ebaa62171b38cb37c3dc420d9b16b/custom_schemas/custom_memory_region.yml#L86-L91).

Reply:
@gabriellandau should we change the keys to reflect the fullname? 

@paul-tavares perhaps if we use the abbreviation everywhere it might come more confusing if we use the fullname in some place and the short in other. Perhaps better add a comment to explain the full meaning?

Reply:
> should we change the keys to reflect the fullname?

I don't think it's necessary.  `pe` is an [ECS field](https://www.elastic.co/guide/en/ecs/current/ecs-pe.html).  I was following that convention with these other field names.